lib/rails/generators/anycable/setup/local/local_generator.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+module AnyCableRailsGenerators
+  module Setup
+    # Generator to help set up AnyCable locally
+    class LocalGenerator < ::Rails::Generators::Base
+      SERVER_VERSION = "v0.6.4"
+      OS_NAMES = %w[linux darwin freebsd win].freeze
+      CPU_NAMES = %w[amd64 arm64 386 arm].freeze
+
+      namespace "anycable:setup:local"
+      source_root File.expand_path("templates", __dir__)
+
+      def server
+        answer = nil
+
+        until [1, 2, 3].include?(answer.to_i)
+          answer = ask "How do you want to install AnyCable-Go WebSocket server? (1) Skip, (2) Homebrew, (3) Download binary"
+        end
+
+        case answer.to_i
+        when 1
+          say_status :help, "🛈 You can read about how to install AnyCable WebSocket here 👉 https://docs.anycable.io/#/anycable-go/getting_started", :yellow
+          return
+        when 2
+          run "brew install anycable-go", abort_on_failure: true
+          run "anycable-go -v", abort_on_failure: true
+        when 3
+          return unless download_binary
+        else
+          raise ArgumentError, "Unknown answer: #{answer}"
+        end
+
+        say_status :info, "✔ AnyCable-Go WebSocket server has been installed"
+      end
+
+      def configs
+        inside("config") { template "cable.yml" }
+      end
+
+      def cable_url
+        environment(nil, env: :development) do
+          'config.action_cable.url = ENV.fetch("CABLE_URL", "ws://localhost:3334/cable").presence'
+        end
+
+        environment(nil, env: :production) do
+          'config.action_cable.url = ENV["CABLE_URL"].presence'
```

Comment:
Maybe, `ENV.fetch("CABLE_URL")`? To make it fail on boot if no env configured.

Or even `ENV.fetch("CABLE_URL") { raise "Please, add CABLE_URL env variable pointing to your AnyCable-Go server" }`

Reply:
Before I wanted, but later I realized that I cannot create a Heroku review app, because it disallows to fill in an empty environment variable. And we want use the async adapter with empty url.